infra/gcb/build.py

```diff
@@ -199,8 +199,19 @@ def get_build_steps(project_yaml, dockerfile_path):
               # We also remove /work and /src to save disk space after a step.
               # Container Builder doesn't pass --rm to docker run yet.
               'rm -r /out && cd /src && cd {1} && mkdir -p {0} && compile && rm -rf /work && rm -rf /src'.format(out, workdir),
-              ],
-            },
+            ],
+          },
+          # test binaries
+          {'name': 'gcr.io/oss-fuzz-base/base-runner',
```

Comment:
Ah, I misremembered and thought that you added a `bad_build_check` step here. Since this changes the behaviour of our builds (and we have no notification system here yet), could you please remove this and add it in your local build.py when you're testing?